### PR TITLE
json logging

### PR DESCRIPTION
**Pull request by aramhakobyan**
customize log fields and having it in the desired format in the kibana/logstash,

### CODE REVIEW

File: nixy.go [hunk 1]
```diff
@@ -5,14 +5,12 @@ import (
 	"flag"
 	"fmt"
 	"io/ioutil"
-	"log"
 	"net/http"
 	"os"
 	"sync"
 	"time"
-
+	"github.com/aramhakobyan/nixy/logging"
```
Comment (schneidexe) [inline nixy.go]: Can you try it with:
```
applog "github.com/zooplus/golang-logging"
```

File: nixy.go [hunk 2]
```diff
@@ -100,9 +99,9 @@ func newHealth() Health {
 }
 
 func nixy_reload(w http.ResponseWriter, r *http.Request) {
-	logger.WithFields(logrus.Fields{
-		"client": r.RemoteAddr,
-	}).Info("marathon reload triggered")
+
+	logger.Infof("marathon reload triggered: client", r.RemoteAddr)
```
Comment (schneidexe) [inline nixy.go]: I think some %s or %v is missing here

File: logging/json_logger.go [hunk 3]
```diff
@@ -0,0 +1,118 @@
+package logging
```
Comment (schneidexe) [inline logging/json_logger.go]: this file can be entirly droppen when using import of github.com/zooplus/golang-logging

File: README.md [hunk 4]
```diff
@@ -144,3 +144,8 @@ You will need the latest NGINX Open Source built with the --with-stream configur
 ### Nagios Monitoring
 
 In case you want to monitor nixy using Nagios (or compatible monitoring) you can use the included `check_nixy` plugin.
+
+### Logging
+
+In order to be able to customize log fields and having it in the desired format in the kibana/logstash, separate package has been added include "github.com/aramhakobyan/nixy/logging" (logging/json_logger.go).
```
Comment (schneidexe) [inline README.md]: Think we can reduce this to reference to github.com/zooplus/golang-logging and add details of purpose there.

**Comment by aramhakobyan** [comment]
@schneidexe  Done, 
please review

**Pull request merged**